docs/Reference/Rest_API/Rest.md

```diff
@@ -61,6 +61,23 @@ You can also use tools such as [Postman] or [cURL] to interact with Teku APIs.
         }
         ```
 
+### Enable the validator REST API
+
+The [validator REST API](https://consensys.github.io/teku/#tag/Validator-Required-Api) is a subset of API methods that are enabled separately from the other REST APIs.
```

Comment:
I'm not sure it's accurate to say they're a subset - they're a completely distinct set of APIs.  The key difference being between REST APIs for the beacon node and these APIs for the validator client.